### DISCUSSION TITLE
Wrong Documentation and want to customize daisyUI tabs.

### DISCUSSION THREAD

**Discussion by cooljameswei** [saadeghi/daisyui#1820] [Q&A] [answered]
https://react.daisyui.com/?path=/story/navigation-tabs--default&args=variant:bordered;dataTheme:

Through documentation it's saying:

```
  const [tabValue, setTabValue] = React.useState(0)
  return (
    <Tabs {...args} value={tabValue} onChange={setTabValue}>
      <Tab value={0}>Tab 1</Tab>
      <Tab value={1}>Tab 2</Tab>
      <Tab value={2}>Tab 3</Tab>
    </Tabs>
  )
```
But there's an error and I update the code to avoid errors.

```
            <Tabs value={tabValue} onChange={setTabValue} boxed={true}>
                <Tabs.Tab value={0} className="btn btn-ghost mt-10 ml-10 mr-10">ongoing</Tabs.Tab>
                <Tabs.Tab value={1} className="btn btn-ghost">completed</Tabs.Tab>
            </Tabs>
```

There isn't exact documentation like `flowbite` how to implment tabs in Daisy UI in detail
https://flowbite-react.com/tabs

Through this link we can easily use FlowBite react tabs. 

And is there any way to change the `tabs.tab` bg  color after I click the tabs.
It sets as `purple` as default.
![image](https://github.com/saadeghi/daisyui/assets/110060055/2020658c-cd49-4065-8e80-deef69438489)

I really want to implement this through daisyUI!

Thanks.

**Comment by saadeghi** [comment]
This repo is for daisyUI (the CSS package).  
Here is the repo for `react-daisyui` (the community project which provides React components with daisyUI styles):  
https://github.com/daisyui/react-daisyui  

Please open an issue there.

**Comment by cooljameswei** [comment, marked as answer]
Thanks for your quick response.
I just implement this and what I was trying to say is that is there's a solution to implement ` daisyui` tabs like `flowbite`.

**Reply by saadeghi** [reply]
Thank you. But since the React components are a community project and I didn't work on it myself, I don't really know why they decided the current approach.  
Hopefully moving this discussion to the React repo will be helpful and leads to improvements.
